modules/i18n/src/translations/ta-IN/portals/myaccount.ts

```diff
@@ -676,6 +676,33 @@ export const myAccount: MyAccountNS = {
                 }
             }
         },
+        "loginVerifyData": {
+            "heading": "உங்கள் உள்நுழைவைச் சரிபார்க்கப் பயன்படுத்தப்படும் தரவு",
+            "description": "உள்நுழைவின் போது உங்கள் அடையாளத்தை மேலும் சரிபார்க்க இந்தத் தரவு பயன்படுத்தப்படுகிறது",
+            "typingdna": {
+                "heading": "தட்டச்சு டி.என்.ஏ தட்டச்சு வடிவங்கள்",
```

Comment:
Shall we use Typing DNA as it is here since that's a product name?

Reply:
Yes, Brand name should not be translated. Should be fixed in other instances as well.